packages/firebase_core/firebase_core_platform_interface/lib/src/platform_interface/platform_interface_firebase_plugin.dart

```diff
@@ -21,6 +21,14 @@ abstract class FirebasePluginPlatform extends PlatformInterface {
   /// requires that the value is synchronously available for use after initialization.
   static Map<dynamic, dynamic> _constantsForPluginApps = {};
 
+  static void setConstantsForPluginApps(String appName,
```

Comment:
For consistency, the method channels should probably use this method instead of directly using the private static?